data/transaction.go

```diff
@@ -118,24 +118,25 @@ type TransactionsPool struct {
 
 // TransactionWithOrder defines a wrapper over transaction
 type TransactionWithOrder struct {
```

Comment:
TransactionWithFeesAndOrder ?

Reply:
I've added feeInfo only for some testing, forgot to remove it, will handle FeeInfo in a separate PR

Reply:
removed feeInfo field